src/Umbraco.Web.BackOffice/Controllers/MediaController.cs

```diff
@@ -47,6 +47,7 @@ namespace Umbraco.Cms.Web.BackOffice.Controllers;
 [PluginController(Constants.Web.Mvc.BackOfficeApiArea)]
 [Authorize(Policy = AuthorizationPolicies.SectionAccessMedia)]
 [ParameterSwapControllerActionSelector(nameof(GetById), "id", typeof(int), typeof(Guid), typeof(Udi))]
+[ParameterSwapControllerActionSelector(nameof(GetByIds), "ids", typeof(int), typeof(Guid), typeof(Udi))]
```

Comment:
Shouldn't the supported types be arrays, similar to `ContentController`?

```suggestion
[ParameterSwapControllerActionSelector(nameof(GetByIds), "ids", typeof(int[]), typeof(Guid[]), typeof(Udi[]))]
```

Reply:
Yes, indeed they should 😁
It was a copy from `GetById` method.